.devcontainer/devcontainer.json

```diff
@@ -8,20 +8,24 @@
 		"args": { "NODE_VERSION": "12" }
 	},
 
+    // Invoke 'nvm' to install our preferred version of node, per the '.nvmrc' file
+    // at the root of the ${workspaceFolder}.
+    "postCreateCommand": ". /usr/local/share/nvm/nvm.sh; nvm install",
+
+    // Expose 'docker.sock' to the dev container and proxy access for the 'node' user.
 	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
+	"postStartCommand": "/usr/local/share/docker-init.sh",
 
 	// Set *default* container specific settings.json values on container create.
 	"settings": { "terminal.integrated.shell.linux": "/usr/bin/zsh" },
 
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
         "dbaeumer.vscode-eslint",
+        "ms-azuretools.vscode-docker",
```

Comment:
Automatically install the vscode-docker extension, otherwise you'll get popups each time you open a Dockerfile.